src/main/java/de/dhbw/karlsruhe/domain/services/SudokuValidatorService.java

```diff
@@ -34,10 +33,56 @@ public boolean isSudokuFieldValid(int[][] sudoku, int row, int col, int value) {
         return true;
     }
 
-    public boolean isSudokuFinished(int[][] gameField) {
+    public boolean isSudokuValid(int[][] sudoku){
+
+        for(int i=0; i<9;i++){
+            HashSet<Integer> row = new HashSet<>();
```

Comment:
Evtl. deklaration aus der for scheife rausziehen damit nicht jedes mal ein neues Set erzeugt wird sondern das gleiche überschreiben?

Reply:
Falls das so Sinn macht

Reply:
Hab es extra in die for loop mit rein gemacht, da man sonst das set nach jedem Schleifendurchlauf wieder leeren muss. Dachte, dann ist neu erstellen genau so gut :)